Assets/Scripts/PlayerController.cs

```diff
@@ -7,11 +7,17 @@ public class PlayerController : MonoBehaviour
 {
     [SerializeField]
     private Rigidbody2D rigidBody;
+    [SerializeField]
+    private float movementSpeed = 5f;
```

Comment:
[Curious] Will this value ever change? If not consider making it a constant value;
[Curious] Why is this a SerializeField type?